src/lib/views/account-settings/SigninSettings.svelte

```diff
@@ -42,12 +42,11 @@
   });
 
   const getLatestVerified = async () => {
-   return isUserVerified = await store.isUserVerified();
-   
+    return (isUserVerified = await store.isUserVerified());
```

Comment:
I don't see isUserVerified defined anywhere. In general we should not do assignments and returns on a single statement or in the same function. We can either do refreshLatestVerfied and refresh the state or we should just return the store value and use caller site to update the variable.

Reply:
This is owing to reactive syntax of Svelte. I just learnt this.